.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,5 +19,4 @@
 - [ ] This code does not rely on migrations in the same Pull Request
 - [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
 - [ ] API release notes have been updated if necessary
-- [ ] This code does not rely on the addition/removal of Azure config environment variables in the same Pull Request
```

Comment:
I think we can keep this point but remove the `Azure` keyword as the same applies even in PaaS?

Reply:
I don't think its applicable for the current deployment process.

Reply:
ah oops misread yep 👍